catalogs/hadoop-compatible-common/src/main/java/com/datastrato/gravitino/hadoop/authentication/AuthenticationConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.hadoop.authentication;
+
+import com.datastrato.gravitino.Config;
+import com.datastrato.gravitino.config.ConfigBuilder;
+import com.datastrato.gravitino.config.ConfigConstants;
+import com.datastrato.gravitino.config.ConfigEntry;
+import java.util.Map;
+
+public class AuthenticationConfig extends Config {
+
+  public static final String IMPERSONATION_ENABLE_KEY = "impersonation-enable";
+
+  public static final boolean DEFAULT_IMPERSONATION_ENABLE = false;
+
+  public static final ConfigEntry<Boolean> CHECK_INTERVAL_SEC_ENTRY =
```

Comment:
I don't think we should share the configuration for Hive and Hadoop.

Reply:
Actually they are different config options, they control different entities. They should be supported in different versions. Some config option may deprecated for the change of underlying systems. They are different.

Reply:
Can someone else assist in leaving comments here? In all, I would like to share some common configuration keys between Hive and HDFS, maybe it's not very proper to make them all shareable. 

@FANNG1 @diqiu50

Reply:
could you provide more context?